lib/linguist/languages.yml

```diff
@@ -2716,6 +2716,20 @@ Kaitai Struct:
   - ".ksy"
   tm_scope: source.yaml
   language_id: 818804755
+KakouneScript:
+  type: programming
+  color: "#6f8042"
+  tm_scope: source.kakscript
+  aliases:
+  - kak
+  - kakscript
+  extensions:
+  - ".kak"
+  filenames:
+  - kakrc
+  - kakrc.local
```

Comment:
Please remove this line as popularity is still too low for inclusion right now.

```suggestion
```

Once this PR has been merged, peeps can use an override for these files until popularity reaches high enough levels for inclusion.